x-pack/metricbeat/module/gcp/_meta/kibana/7/lens/057de170-e88d-11ea-bf8c-d13ebf358a78.json

```diff
@@ -14,7 +14,7 @@
                                     "dataType": "number",
                                     "isBucketed": false,
                                     "label": "Maximum of gcp.billing.total",
-                                    "operationType": "max",
+                                    "operationType": "sum",
```

Comment:
If we change the aggregation type, we should also update the label accordingly (now it's named "Maximum of [..]").

But I am mostly interested in the change of aggregation type.

Reply:
Oh yeah I forgot about the label. Previously, with only one document per day, we used `max` to display the cost. However, since the data granularity has increased, generating more documents daily, we need to use `sum` to accurately aggregate the costs from all documents.

Reply:
So now we have multiple documents per day, instead of just one. Got it.

We should then update the PR description to reflect this change. The PR description should contain a TL;DR summary of all the changes happening.